cluster-stamp.bicep

```diff
@@ -0,0 +1,1784 @@
+
+var agwName = 'apw-${clusterName}'
+
+var aksIngressDomainName = 'aks-ingress.${domainName}'
+var aksBackendDomainName = 'bu0001a0008-00.${aksIngressDomainName}'
+var policyResourceIdAKSLinuxRestrictive = '/providers/Microsoft.Authorization/policySetDefinitions/42b8ef37-b724-4e24-bbc8-7a7708edfe00'
+var policyResourceIdEnforceHttpsIngress = '/providers/Microsoft.Authorization/policyDefinitions/1a5b4dca-0b6f-4cf5-907c-56316bc1bf3d'
+var policyResourceIdEnforceInternalLoadBalancers = '/providers/Microsoft.Authorization/policyDefinitions/3fc4dc25-5baf-40d8-9b05-7fe74c1bc64e'
+var policyResourceIdRoRootFilesystem = '/providers/Microsoft.Authorization/policyDefinitions/df49d893-a74c-421d-bc95-c663042e5b80'
+var policyResourceIdEnforceResourceLimits = '/providers/Microsoft.Authorization/policyDefinitions/e345eecc-fa47-480f-9e88-67dcc122b164'
+var policyResourceIdEnforceImageSource = '/providers/Microsoft.Authorization/policyDefinitions/febd0533-8e55-448f-b837-bd0e06f16469'
+var policyResourceIdEnforceDefenderInCluster = '/providers/Microsoft.Authorization/policyDefinitions/a1840de2-8088-4ea8-b153-b4c723e9cb01'
+var policyAssignmentNameAKSLinuxRestrictive = guid(policyResourceIdAKSLinuxRestrictive, resourceGroup().name, clusterName)
+var policyAssignmentNameEnforceHttpsIngress = guid(policyResourceIdEnforceHttpsIngress, resourceGroup().name, clusterName)
+var policyAssignmentNameEnforceInternalLoadBalancers = guid(policyResourceIdEnforceInternalLoadBalancers, resourceGroup().name, clusterName)
+var policyAssignmentNameRoRootFilesystem = guid(policyResourceIdRoRootFilesystem, resourceGroup().name, clusterName)
+var policyAssignmentNameEnforceResourceLimits = guid(policyResourceIdEnforceResourceLimits, resourceGroup().name, clusterName)
+var policyAssignmentNameEnforceImageSource = guid(policyResourceIdEnforceImageSource, resourceGroup().name, clusterName)
+var policyAssignmentNameEnforceDefenderInCluster = guid(policyResourceIdEnforceDefenderInCluster, resourceGroup().name, clusterName)
+var isUsingAzureRBACasKubernetesRBAC = (subscription().tenantId == k8sControlPlaneAuthorizationTenantId)
+
+resource alaRgRecommendations 'Microsoft.Insights/activityLogAlerts@2020-10-01' = {
+  name: 'AllAzureAdvisorAlert'
+  location: 'Global'
+  properties: {
+    scopes: [
+      resourceGroup().id
+    ]
+    condition: {
+      allOf: [
+        {
+          field: 'category'
+          equals: 'Recommendation'
+        }
+        {
+          field: 'operationName'
+          equals: 'Microsoft.Advisor/recommendations/available/action'
+        }
+      ]
+    }
+    actions: {
+      actionGroups: []
+    }
+    enabled: true
+    description: 'All azure advisor alerts'
+  }
+  dependsOn: []
+}
+
+resource ssPrometheusAll 'Microsoft.OperationalInsights/workspaces/savedSearches@2020-08-01' = {
+  name: '${logAnalyticsWorkspaceName}/AllPrometheus'
+  properties: {
+    etag: '*'
```

Comment:
I had already tried moving `etag` where the docs indicates (one level above) without luck. It throws an error. 

I tried once today and checked for new apis. Same result.

Reply:
Sounds good.  Probably just a bicep tooling issue.  I know it's a valid property, and necessary to prevent conflict responses on redeploy... so, /shrug.  Thanks for checking!  Marking resolved.

Reply:
good catch @ckittel 

done | addressed from 5f7b0b49bedd4dc5ebde6767e26471ccc04318a5